_posts/2021-06-19-release-0.9.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Release of v0.9
+---
+
+gfx-rs community's goal is to make graphics programming in Rust easy, fast, and reliable. Our current main projects are:
+
+  - [gfx-rs](https://github.com/gfx-rs/gfx) makes low-level GPU programming portable with low overhead. It's a single Vulkan-like Rust API with multiple backends that implement it: Direct3D 12/11, Metal, Vulkan, and even OpenGL ES.
+  - [naga](https://github.com/gfx-rs/naga) translates the shaders between languages, including WGSL. Also provides validation and processing facilities on the intermediate representation.
+  - [wgpu](https://github.com/gfx-rs/wgpu) is built on top of gfx-rs and [gpu-alloc](https://github.com/zakarumych/gpu-alloc)/[gpu-descriptor](https://github.com/zakarumych/gpu-descriptor). It provides safety, accessibility, and strong portability of applications.
```

Comment:
note: for the future entries, we should probably be less vocal about gpu-alloc/gpu-descriptor, since they are just backend details now with wgpu-hal.
But for this release it's definitely spelled right!